doc/user/content/release-notes.md

```diff
@@ -51,6 +51,12 @@ Wrap your release notes at the 80 character mark.
 - Commit consumed offsets to Kafka to facilitate source ingest progress
   monitoring.
 
+- Fix a bug that would cause `DROP` statements targeting multiple objects to fail
+  when those objects had dependent objects in common {{% gh 5316 %}}.
+
+- Fix a bug that would allow `CREATE OR REPLACE` statements to create dependencies
```

Comment:
For variety, this could be rephrased to start with `Prevent...`––totally optional!